src/ansys/fluent/core/quantity.py

```diff
@@ -0,0 +1,562 @@
+        "Pa": "N m^-2",
+        "W": "N m s^-1",
+        "J": "N m",
+        "V": "A ohm",
+        "F": "N m V^-2",
+        "H": "N m A^-2",
+        "S": "ohm^-1",
+        "Wb": "N m A^-1",
+        "T": "Wb m^-2",
+        "dyne": "g cm s^-2",
+        "erg": "dyne cm",
+        "pdl": "lbm ft s^-2",
+        "BTU": "J",
+        "psi": "lbf in^-2",
+        "lbf": "slug ft s^-2",
+        "psf": "lbf ft^-2",
+        "ohm": "kg m^2 s^-3 A^-2",
+        "Hz": "s^-1",
+    }
+
+    _derived_units_with_conversion_factor = {
+        "l": (0.001, "m^3"),
+        "gal": (0.0037854117839999993, "m^3"),
+        "BTU": (1055.056, "J"),
+    }
+
+    _multipliers = {
+        "d": 10**-1,
+        "c": 10**-2,
+        "m": 10**-3,
+        "μ": 10**-6,
+        "n": 10**-9,
+        "p": 10**-12,
+        "f": 10**-15,
+        "a": 10**-18,
+        "z": 10**-21,
+        "y": 10**-24,
+        "da": 10**1,
+        "h": 10**2,
+        "k": 10**3,
+        "M": 10**6,
+        "G": 10**9,
+        "T": 10**12,
+        "P": 10**15,
+        "E": 10**18,
+        "Z": 10**21,
+        "Y": 10**24,
+    }
+
+    _dimension_order = OrderedDict(
+        [
+            ("Mass", "M"),
+            ("Length", "L"),
+            ("Time", "T"),
+            ("Temperature", "K"),
+            ("Angle", "Angle"),
+            ("ChemicalAmount", "mol"),
+            ("Light", "cd"),
+            ("Current", "A"),
+            ("SolidAngle", "sr"),
+            ("", ""),
+        ]
+    )
+
+    _conversion_map = {
+        "kg": 1,
+        "g": 0.001,
+        "lb": 0.45359237,
+        "lbm": 0.45359237,
+        "slug": 14.59390293720637,
+        "m": 1,
+        "cm": 0.01,
+        "ft": 0.30479999999999996,
+        "in": 0.0254,
+        "s": 1,
+        "A": 1,
+        "mol": 1,
+        "cd": 1,
+        "sr": 1,
+        "radian": 1,
+        "degree": 0.017453292519943295,
+        "K": 1,
+        "C": 1,
+        "F": 0.5555555555555556,
+        "R": 0.5555555555555556,
+    }
+
+    _si_map = {
+        "kg": "kg",
+        "g": "kg",
+        "lb": "kg",
+        "lbm": "kg",
+        "slug": "kg",
+        "m": "m",
+        "cm": "m",
+        "ft": "m",
+        "in": "m",
+        "s": "s",
+        "A": "A",
+        "mol": "mol",
+        "cd": "cd",
+        "sr": "sr",
+        "radian": "radian",
+        "degree": "radian",
+        "K": "K",
+        "C": "K",
+        "F": "K",
+        "R": "K",
+        "": "",
+    }
+
+    _offset_conversions = {
+        "K": 1,
+        "C": 274.15,
+        "F": 255.92777777777778,
+        "R": 0.5555555555555556,
+    }
+
+    _offset_dict = {
+        "K": {"C": 274.15, "F": 255.92777777777778, "R": 0.5555555555555556},
+        "C": {"K": -272.15, "F": -17.2222222222222, "R": -272.59444444444443},
+        "F": {"C": 33.79999999999993, "K": -457.87, "R": -458.67},
+        "R": {"C": 493.4699999999999, "F": 460.66999999999996, "K": 1.7999999999999998},
+    }
```

Comment:
Do we want to store the above in a separate file?

Reply:
Not sure yet. Needs discussion about handling this from and to individual temperature conversion corner cases. Thank you.